<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerPictureWidgetBase</class>
 <widget class="QWidget" name="QgsComposerPictureWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>614</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Picture Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>329</width>
        <height>612</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout">
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="mPreviewGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Main properties</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Path</string>
            </property>
            <property name="buddy">
             <cstring>mPictureLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="mPictureLineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="mPictureBrowseButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>32767</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="mSearchDirectoriesGroupBox">
         <property name="title">
          <string>Search directories</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="mPreviewListWidget">
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragDrop</enum>
            </property>
            <property name="movement">
             <enum>QListView::Free</enum>
            </property>
            <property name="flow">
             <enum>QListView::LeftToRight</enum>
            </property>
            <property name="isWrapping" stdset="0">
             <bool>true</bool>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
            <property name="gridSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="viewMode">
             <enum>QListView::IconMode</enum>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mSearchDirectoriesComboBox"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="mRemoveDirectoryButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mAddDirectoryButton">
              <property name="text">
               <string>Add...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Rotation</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <item row="1" column="0">
           <widget class="QCheckBox" name="mRotationFromComposerMapCheckBox">
            <property name="text">
             <string>Sync with map</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="mComposerMapComboBox"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QDoubleSpinBox" name="mRotationSpinBox">
            <property name="suffix">
             <string> Â°</string>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBoxBasic</class>
   <extends>QGroupBox</extends>
   <header location="global">qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mSearchDirectoriesComboBox</tabstop>
  <tabstop>mPictureLineEdit</tabstop>
  <tabstop>mPictureBrowseButton</tabstop>
  <tabstop>mRotationFromComposerMapCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
